project/app/src/main/java/com/nexte/nexte/ShowProfileScene/ShowProfileWorker.kt

```diff
@@ -4,39 +4,36 @@ package com.nexte.nexte.ShowProfileScene
  * Created by albino on 25/03/18.
  */
 
+/* This class verify if the user is validad
+    and return the user as response */
+
 class ShowProfileWorker {
 
     constructor() { }
 
-    fun getUserProfile(request: ShowProfileModel.Request, completion: (ShowProfileModel.Response) -> Unit) {
+    fun getUserProfile(request: ShowProfileModel.Request,
+                       completion: (ShowProfileModel.Response) -> Unit) {
+
         val username = request.username
         val tokenID = request.tokenID
 
-        var facebookUsername: String = ""
-        var pictureAddress: String = ""
-        var rankingPosition: Int = -1
-        var team: String = ""
-        var teamLocation: String = ""
-        var age: Int? = -1
-
-        if(tokenID.equals("")){
-            facebookUsername = ""
-            pictureAddress = ""
-            rankingPosition = -1
-            team = ""
-            teamLocation = ""
-            age = -1
+        val empty = ShowProfileModel.Player("", -1, "", "",
+                                             "", "", -1)
+
+        val userTest = ShowProfileModel.Player("gabrielalbino", 2,
+                                                "imgur.com/nudh486d4",
+                                                "enggabriel@gmail.com", "masculino",
+                                                "ASCAD", 19)
+        var returned: ShowProfileModel.Player? = null
+
```

Comment:
Aqui foram as únicas coisas que não gostei no código de vocês. Colocar uma variável com nome "userTest" não é legal, sei que os dados ainda estão mockados e não vem de uma API mas mesmo assim, colocar um nome desse vai forçar vocês a refatorarem depois.

Uma dica é tentar usar sempre nomes mais significativos possíveis, por exemplo:
empty poderia ser emptyUser
returned poderia ser returnedUser

Enfim, são apenas dicas mesmo.

Reply:
Mudanças realizadas